source/_integrations/tuya.markdown

```diff
@@ -118,7 +118,7 @@ Click the created project to enter the `Project Overview` page and get the `Auth
   description: >
     - App account not linked with cloud project: On the [Tuya IoT Platform](https://iot.tuya.com/cloud/), you have linked devices by using Tuya Smart or Smart Life app in your cloud project. For more information, see [Link devices by app account](https://developer.tuya.com/en/docs/iot/Platform_Configuration_smarthome?id=Kamcgamwoevrx&_source=7a356dd493196a01bb9021b7680a2a45#title-3-Link%20devices%20by%20app%20account).
 
-    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields. Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
+    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which `Tuya Smart` allows, may be avoided. See [community.home-assistant.io#353128](https://community.home-assistant.io/t/new-tuya-integration-2021-11-unable-to-log-in/353128/43)). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
```

Comment:
We should not refer to community guides. If this documentation needs to be improved, we should add it directly here and not link people around.

Reply:
Sure, I have no strong opinion here so removed the reference.